src/pages/lessons/[slug].tsx

```diff
@@ -397,14 +396,16 @@ const Lesson: React.FC<LessonProps> = ({
         console.debug(
           `changed to viewing isFullscreen: ${isFullscreen} mediaPresent: ${mediaPresent}`,
         )
+        if (!initialMediaPresent && mediaPresent) {
+          videoService.send({
+            type: 'LOAD_RESOURCE',
+            resource: lesson,
+          })
+        }
```

Comment:
only trigger LOAD_RESOURCE if the initial lesson doesn't include media urls and the full lesson does.